app/services/model/RejectionReason.scala

```diff
@@ -44,4 +46,22 @@ object RejectionReason {
 
     errors.map(RejectionReason.apply _)
   }
+
+  def getErrorDescription(errorCode: String): String =
+    allRejectedErrors.find(_.code == errorCode).map(_.description).getOrElse("Unknown error")
+
+  def fromNotifications(notifications: Seq[Notification]): Seq[RejectionReason] = {
+
+    val rejectionNotification = notifications.filter(_.functionCode == SubmissionStatus.Rejected.fullCode).headOption
```

Comment:
find

Reply:
I want to avoid having Option, but I used `headOption` 🤦‍♂ Fixed